### ISSUE TITLE
.csv

### ISSUE THREAD

**Issue by Wyue1** [HappyColor/SpeechFormer2#5]
Did you fill in the name and label of the self created. csv file you mentioned in readme? What do they refer to respectively

**Comment by HappyColor** [comment]
Yes, you have to fill in the `name` and `label` information (demo can be found [here](https://github.com/HappyColor/SpeechFormer/tree/master/metadata)). `name` represents the name of the feature file (`.mat` file) and `label` is the label of the utterance sample.
